README.md

```diff
@@ -133,15 +147,12 @@ match result {
 - [Lucio Franco](https://github.com/LucioFranco)
 - [Fletcher Haynes](https://github.com/fhaynes)
 - [Timon Post](https://github.com/TimonPost)
-- [Justin LeFebvre](https://github.com/jstnlef)
-
-We want to give credit to [gaffer on games](https://gafferongames.com/) as we have used his guide to building a game networking protocol to build this library. 
```

Comment:
why is this removed?

Reply:
Because we already said in the header of the read me that we had some ideas from him. However, there are millions of other resources we could be referencing here.

Reply:
Ah okay, yeah as long as we give credit to atleast what inspired us id like that.